commons/hadoop/2.7.6/files/entrypoint.sh

```diff
@@ -48,10 +46,11 @@ case $1 in
     # start Hadoop daemons
     $HADOOP_PREFIX/sbin/hadoop-daemon.sh --config $HADOOP_PREFIX/etc/hadoop --script hdfs start datanode
     $HADOOP_PREFIX/sbin/yarn-daemon.sh --config $HADOOP_PREFIX/etc/hadoop start nodemanager
-    # keep container running
-    tail -f /var/log/dmesg
     ;;
   *)
     echo "$USAGE"
     ;;
 esac
+
+# keep container running and show logs
+tail -f $HADOOP_LOG_DIR/*
```

Comment:
To 
*  make the container's logs meaningful 
*  and also to make it compatible with the current ubuntu 18.04 image which has a delay in creating /var/log/dmesg file which may lead the container to crash